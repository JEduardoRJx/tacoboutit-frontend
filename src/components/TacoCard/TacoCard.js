import React, { Component } from 'react'
import { Text, View, Image, StyleSheet, Dimensions } from 'react-native'
const checkMarkImg = require('../../../assets/check-mark.png');

export default class TacoCard extends Component {
  render() {
    const { type } = this.props;
    return (
      <View style={styles.card}>
        <Text>{type}</Text>
        <Image 
          source={checkMarkImg}
          style={{ width: 30, height: 30, }}
        />
      </View>
    )
  }
}

const styles = StyleSheet.create({
  card: {
    width: Dimensions.get('window').width * .75,
    flexDirection: 'row',
    justifyContent: 'space-between',
    alignSelf: 'center',
    marginVertical: 10,
    backgroundColor: 'white',
    padding: 10,
    alignItems: 'center',
    borderRadius: 10,
  }
})