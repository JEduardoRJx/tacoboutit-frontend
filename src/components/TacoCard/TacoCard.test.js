import React from 'react';
import renderer from 'react-test-renderer';

import TacoCard from '../TacoCard/TacoCard';

describe('TacoCard', () => {
  const mockType = 'Carne Asada'
  it('renders correctly', () => {
    const wrapper = renderer.create(<TacoCard type={mockType} />).toJSON();
    expect(wrapper).toMatchSnapshot();
  });
});