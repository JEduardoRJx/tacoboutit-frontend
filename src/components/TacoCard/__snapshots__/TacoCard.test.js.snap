// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TacoCard renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "alignSelf": "center",
      "backgroundColor": "white",
      "borderRadius": 10,
      "flexDirection": "row",
      "justifyContent": "space-between",
      "marginVertical": 10,
      "padding": 10,
      "width": 562.5,
    }
  }
>
  <Text>
    Carne Asada
  </Text>
  <Image
    source={1}
    style={
      Object {
        "height": 30,
        "width": 30,
      }
    }
  />
</View>
`;
